metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Dyanmically generates /etc/ssh/ssh_known_hosts based on search indexes'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '4.1.0'
+version           '4.2.0'
```

Comment:
also please don't bump the version in the metadata, automation typically handles this and it causes merge conflicts